plugins/search/providers/elastic_search/batch/KExportMediaEsearchEngine.php

```diff
@@ -58,7 +57,10 @@ public function fillCsv(&$csvFile, &$data)
 	protected function addHeaderRowToCsv($csvFile, $additionalFields,
 	                                     $mappedFileds = null)
 	{
-		$headerRow = 'EntryID, Name, Description, Tags, Categories, UserID, CreatedAt, UpdatedAt ';
+		$headerRow = 'EntryID, Name, Description, Tags, Categories, UserID, CreatedAt, UpdatedAt, CreatorId ';
+		if($this->getEntryType($additionalFields) == KalturaEntryType::MEDIA_CLIP){
```

Comment:
@TalDubovKaltura  I think it would be better to create a new function "addAdditionalFieldsHeaders", calculate the headerRow addition in it, and then concat the function result.

Reply:
@hilak Totally agree with that, method has been added.